test/xref2/v408_and_above/github_issue_587.t/run.t

```diff
@@ -1,5 +1,25 @@
 A quick test to repro the issue found in #587
 
   $ ./build.sh
+  File "odoc_bug__a_intf.cmt":
```

Comment:
What's going on here? I don't understand why this would start failing now?

Reply:
Fixed. I misunderstood lookup by identifier in the environment. The previous code was removing shadowed items from the environment but we need to keep them in the case of aliases to the same name. We only need to remove them from "lookup by name" results.